apcd_cms/src/apps/utils/apcd_database.py

```diff
@@ -152,6 +152,44 @@ def get_user_role(user):
         if conn is not None:
             conn.close()
 
+def get_submitter_users():
+    cur = None
+    conn = None
+    try:
+        conn = psycopg.connect(
+            host=APCD_DB['host'],
+            dbname=APCD_DB['database'],
+            user=APCD_DB['user'],
+            password=APCD_DB['password'],
+            port=APCD_DB['port'],
+            sslmode='require'
+        )
+        query = """
+        SELECT DISTINCT submitter_users.submitter_id,
+        submitter_users.user_id,
+        submitter_users.user_number,
+        users.user_email,
+        users.user_name,
+        submissions.payor_code
+        FROM submitter_users
+        JOIN users
+           ON submitter_users.user_id = users.user_id
+           AND submitter_users.user_number = users.user_number
+        JOIN submissions
+            ON submitter_users.submitter_id = submissions.submitter_id;
```

Comment:
having a order by helps the listing view consistent on refreshes
```suggestion
            ON submitter_users.submitter_id = submissions.submitter_id
            ORDER BY users.user_id;
```